plugins/woocommerce-admin/client/products/fields/enriched-label.js

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
Curious why you decided to write this component in `js` versus `tsx`?

Reply:
I thought I have transformed this file to TS, but I didn't. Sorry for that; I converted this file to TS in commit afb14b4